rpc/websockets.go

```diff
@@ -33,6 +34,17 @@ import (
 	evmtypes "github.com/evmos/ethermint/x/evm/types"
 )
 
+const (
+	wsReadBuffer       = 1024
+	wsWriteBuffer      = 1024
+	wsPingInterval     = 60 * time.Second
+	wsPingWriteTimeout = 5 * time.Second
+	wsPongTimeout      = 30 * time.Second
+	wsMessageSizeLimit = 15 * 1024 * 1024
```

Comment:
I feel this should be server config options instead of constants

Reply:
ah, so!

I did it this way, because Ethereum does -- wdyt?

Reply:
let's add them here: https://github.com/evmos/ethermint/tree/main/server/config

Reply:
ah this is a very sensible place for that....